File: Backend/TN.TNM.DataAccess/Models/MobileAppConfiguration/CauHinhAnhLinkWebModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TN.TNM.DataAccess.Models.MobileAppConfiguration
{
    public class CauHinhAnhLinkWebModel
    {
        public Guid? Id { get; set; }
        public string Anh { get; set; }
        public string Link { get; set; }
        public int? Type { get; set; }
    }
}

File: Fontend/TNT.N8.Web/TNT.N8.Web/ClientApp/src/app/shared/models/CauHinhAnhLinkWebModel.ts
export class CauHinhAnhLinkWebModel {
    id: string | null;
    anh: string;
    link: string;
    type: number;//--1: Banner, 2: Ưu đãi, 3: quảng cáo trái, 4: quảng cáo phải
    constructor(type: number) {
        this.type = type;
    }
}